apps/astarte_appengine_api/test/astarte_appengine_api/device/interface_value_test.exs

```diff
@@ -172,4 +174,201 @@ defmodule Astarte.AppEngine.API.InterfaceValueTest do
     assert InterfaceValue.cast_value(expected_types, object) ==
              {:error, :unexpected_value_type, [expected: :longinteger]}
   end
+
+  property "casting valid datetime values" do
```

Comment:
Properties have usually a complete sentence as name. In this case, something like
```suggestion
  property "Valid Datetime values are correctly casted" do
```
would sound better.
The same goes for other properties below